webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -256,24 +278,22 @@ export class MembraneWebRTC {
         });
       };
 
+      const label =
+        this.participants.find((p) => p.mids.includes(mid))?.displayName || "";
       this.callbacks.onAddTrack?.({
```

Comment:
I think that we have a bug in this place. If we receive two different tracks within the same stream `onAddTrack` will be called twice and frontend will create two audio elements (once for incoming audio track and another for incoming video track).  We are probably mixing too much webrtc logic with business logic, once we are operate on raw tracks and the other time we interpret them as something different. `onAddTrack` should probably stay as it is and frontend should watch out if it is duplicating some video/audio elements.

Reply:
We no longer create AudioElements. Here we only notify about new incoming track. Frontend will create one VideoElement per stream.